xtable-core/src/main/java/org/apache/xtable/delta/DeltaSchemaExtractor.java

```diff
@@ -41,22 +37,10 @@
 import org.apache.xtable.model.schema.InternalType;
 import org.apache.xtable.schema.SchemaUtils;
 
-/**
- * Converts between Delta and InternalTable schemas. Some items to be aware of:
- *
- * <ul>
- *   <li>Delta schemas are represented as Spark StructTypes which do not have enums so the enum
```

Comment:
I think we can live this file as is right?

Reply:
@vaibhavk1992 let's remove the changes to this file. They don't seem necessary

Reply:
@vaibhavk1992 please follow up on this